### PR TITLE
Adding max-connection parameter to outbound server

### PR DESCRIPTION
**Pull request by kalmik**

### CODE REVIEW

File: greenswitch/esl.py [hunk 1]
```diff
@@ -426,8 +429,15 @@ def listen(self):
         self.server.listen(100)
 
         while True:
+            if self.connection_pool.full():
+                logging.info('Have no available slots to allocate this client connection pool reached %s' %
```
Comment (diasbruno) [inline greenswitch/esl.py]: Message can be `Unable to connect. Reached max. number of connections (max: %s).`.

Comment (kalmik) [reply greenswitch/esl.py]: Thanks

Comment (diasbruno) [reply greenswitch/esl.py]: Correct, @italorossi. So perhaps this can be shorter...
 'Rejecting call, server is at full capacity. (max: %s).'

Comment (iuridiniz) [reply greenswitch/esl.py]: s/have/there are/

Comment (iuridiniz) [reply greenswitch/esl.py]: Excuse me and ignore my last comment

**Pull request merged**